ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -3242,7 +3257,13 @@ MmLoadSystemImage(IN PUNICODE_STRING FileName,
     /* FIXME: Call driver verifier's loader function */
 
     /* Write-protect the system image */
+#if !defined(CORE_16449_IS_FIXED) && DBG
+    DPRINT1("(CORE-16449) Calling MiWriteProtectSystemImage() for %wZ (%wZ)\n",
+            &LdrEntry->BaseDllName, &LdrEntry->FullDllName);
+#endif
+#if defined(CORE_16449_IS_FIXED) || DBG
```

Comment:
Why only calling this on DBG? The code above should work on release builds.

Reply:
'I will never let that protection feature reach any release until [...]' (c) Joachim

Reply:
>> if (wcscmp(LdrEntry->BaseDllName.Buffer, L"ftfd.dll"  ) == 0 || wcscmp(LdrEntry->BaseDllName.Buffer, L"win32k.sys") == 0)
Woah is that horrible! And what about pfmodnt?
Do you really think it's beneficial to add such ugly exception-per-driver-hacks into the kernel?

Reply:
> horrible

??

> And what about pfmodnt?

What is 'pfmodnt'?

> Do you really think it's beneficial to add such ugly exception-per-driver-hacks into the kernel?

On one PR I get complain about adding too much DPRINT1, on another I get complain about not adding enough: usual nonsense, as I see it...

Why don't you submit a better PR to investigate and fix CORE-16449 (for example), rather than saying 'no' so often?

Reply:
> usual nonsense, as I see it...

Do you know what the real nonsense is? I will tell you:

- You have [**53 open PRs**](https://github.com/reactos/reactos/pulls/SergeGautherie) in the repo, is this normal for just one person?
- As a result, all our existing manpower is being ~~used~~ abused for reviewing your mostly low-quality and untested Pull Requests

> Why don't you submit a better PR to investigate and fix

And after all this you even dare to ask somebody to spend even more time? Hopefully some of devs already decided to ignore any PRs and changes done by you, which includes me (_you might have noticed I stopped adding labels to your PRs_), so we still have some manpower to handle PRs from other people.

The more PRs you open, the more I think you do it intentionally to just abuse our manpower and ruin the project. So at this point the best thing you can do to save the project is to **stop sending any PRs**.

After this your comment I really want to kick you from our repository, but I'm not going to close all your PRs because this will result in spamming all devs e-mails, and so I don't recommend anyone doing this including you. Just leave. Thanks for understanding.